pkg/files/filehandling.go

```diff
@@ -46,8 +50,39 @@ func setDefaultConfig() {
 
 // InitFileHandler creates a new file handler for the file backend we want to use
 func InitFileHandler() {
-	fs = afero.NewOsFs()
-	afs = &afero.Afero{Fs: fs}
+	fileType := config.FilesType.GetString()
+	
+	if fileType == "s3" {
+		// Get S3 configuration
```

Comment:
Please refactor this (and the local fs usage) into a new function, then use a switch case to create the handler depending on the config